packages/legend-application-studio/src/components/editor/side-bar/RegisterService.tsx

```diff
@@ -0,0 +1,139 @@
+/**
+ * Copyright (c) 2020-present, Goldman Sachs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { useState } from 'react';
+import { observer } from 'mobx-react-lite';
+import { LEGEND_STUDIO_TEST_ID } from '../../LegendStudioTestID.js';
+import { useEditorStore } from '../EditorStoreProvider.js';
+import {
+  ContextMenu,
+  PanelContent,
+  PURE_ServiceIcon,
+  clsx,
+  PlayIcon,
+  Dialog,
+  Modal,
+  TimesIcon,
+  EmptyWindowRestoreIcon,
+  WindowMaximizeIcon,
+} from '@finos/legend-art';
+import type { BulkServiceRegistrationState } from '../../../stores/sidebar-state/BulkServiceRegistrationState.js';
+import { BulkServiceRegistrationEditor } from '../edit-panel/service-editor/BulkServiceRegistrationEditor.js';
+import { noop } from '@finos/legend-shared';
+
+export const RegisterService = observer(
+  (props: { bulkServiceRegistrationState: BulkServiceRegistrationState }) => {
+    const editorStore = useEditorStore();
+    const services = editorStore.graphManagerState.graph.ownServices;
+    const [open, setOpen] = useState(false);
+
+    const [isMaximized, setIsMaximized] = useState(false);
+    const toggleMaximize = (): void => setIsMaximized(!isMaximized);
+
+    const renderTestables = (): React.ReactNode => (
```

Comment:
lets change this to the correct name. `ServiceItem` and lets also not reuse the class names from testable but create separate css files for the buil service registration